app/src/main/java/com/example/findu/presentation/ui/search/detail/SearchProtectingDetailFragment.kt

```diff
@@ -19,26 +19,42 @@ import com.example.findu.data.mapper.todomain.toDetailSearchRvTag
 import com.example.findu.databinding.FragmentSearchDetailProtectingBinding
 import com.example.findu.domain.model.search.DetailProtectData
 import com.example.findu.presentation.ui.search.viewmodel.DetailSearchViewModel
+import com.naver.maps.geometry.LatLng
+import com.naver.maps.map.CameraUpdate
+import com.naver.maps.map.MapView
+import com.naver.maps.map.NaverMap
+import com.naver.maps.map.overlay.Marker
+import com.naver.maps.map.overlay.OverlayImage
 import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.flow.collectLatest
 import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
 
 @AndroidEntryPoint
 class SearchProtectingDetailFragment : Fragment() {
     private lateinit var binding: FragmentSearchDetailProtectingBinding
-    private var isDetailVisible = false
     private val viewModel by viewModels<DetailSearchViewModel>()
     private var cardId: Long = -1
```

Comment:
코틀린이 암묵적으로 Int -> Long 으로 변환하기 때문에 불필요한 변환을 방지하기 위해 아래와 같이 추천드려용
```suggestion
    private var cardId: Long = -1L
```

Reply:
넵!